README.md

```diff
@@ -19,6 +19,10 @@ that incorporates and documents best practices for a performant and scalable des
 security by default for control, logging and evidence generation. It can be  simply deployed by
 customers through a Terraform workflow.
 
+## Disclaimer
```

Comment:
Consider making this a section with various points using `*` format. 

For instance, is this where we can link to the foundations blueprint for additional good practices?